src/models/Role.ts

```diff
@@ -0,0 +1,11 @@
+import { Schema, model } from 'mongoose';
+
+const Role = new Schema({
+  value: {
+    type: String,
+    unique: true,
+    default: 'USER',
```

Comment:
USER value should be imported from constant.

Reply:
@MarynaBandarenka updated